enos/enos.py

```diff
@@ -205,12 +235,24 @@ def install_os(env=None, **kwargs):
 - Install default network
 
 Options:
+  -e ENV --env=ENV  Path to the environment yaml file. You should
+                    use this option when you wanna link a specific
+                    experiment [default: %s].
   -h --help            Show this help message.
-""")
+  -v --version         Show version number.
+  -vv                  Verbose mode.
+  -s --silent          Quiet mode.
+""" % os.path.join(SYMLINK_NAME, 'env'))
 def init_os(env=None, **kwargs):
     logging.debug('phase[init]: args=%s' % kwargs)
+
+    # Override default result directory name when --env parameter is given
```

Comment:
This piece of code seems duplicated 4 times. Is it possible to factorize it somehow ?